### ISSUE TITLE
MangaDex description lost

### ISSUE THREAD

**Issue by TesterAll** [keiyoushi/tachiyomi-extensions-source#872]
### Source information

MangaDex V1.4.192

### Source language

Multiple

### Steps to reproduce

Check the description of an entry with more than one **-**
Example could be some 18+ manga: some of them have in their description a table with tags.
Tags could be written like this:

| Namespace | TAGS |
| :--- | :--- |

### Expected behavior

I censored some 18+ tags in the image attached.
You should read everything after the | :--- | :--- |
![Screenshot_2024-02-01-01-01-57-622_eu kanade tachiyomi sy-edit](https://github.com/keiyoushi/extensions-source/assets/158376538/b48eb306-0ca6-430a-9ab3-c61422a4682d)

### Actual behavior

![Screenshot_2024-02-01-01-01-00-523_eu kanade tachiyomi sy-edit](https://github.com/keiyoushi/extensions-source/assets/158376538/ce81b864-bd76-4a8d-b1ab-3fa16acd2c70)

### Mihon/Tachiyomi version

TachiyomiSY preview 542/ mihon 0.16.3

### Android version

14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by 0Ha16wA5** [comment]
~~Not a bug.~~

~~Mangadex description is provided by the community, some are using non standard description that different from other entries or simply yet to be added (a.k.a placeholder).~~

~~I tried to visit the exact entry and it seems already updated.~~

**Comment by TesterAll** [comment]
I asked MangaDex team and they said that's not a problem their side, since that markdown `---` or `--` is accepted and it's the extension that has to be updated 🤷‍♂️

**Comment by AwkwardPeak7** [comment]
Tachiyomi/Mihon doesn't support markdown in description so extension filters them out

**Comment by TesterAll** [comment]
Yes, but this problem could be solved if the extension takes the description "as is", just like it does when it's only like `:-`
I think users likes more an ugly to look at description instead of a missing description

**Comment by 0Ha16wA5** [comment]
@AwkwardPeak7  Instead of filtering out, I believe it's better to just display them as is so the tag still can be searchable on the app?

**Comment by AwkwardPeak7** [comment]
then you would have links and whatnot in the description, (which won't render) and clutter the description

**Comment by TesterAll** [comment]
If you add a toggle in settings?
